docs-source/content/userguide/cicd/_index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "CI/CD"
```

Comment:
Reader view comment: When you look at this in context of the website, the title name seem like it should be more along the line of  "CI/CD Processing" to parallel topics at the same level of managing domains, managing operators, etc.

Reply:
done